sdk/formrecognizer/ci.yml

```diff
@@ -0,0 +1,42 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+resources:
```

Comment:
Watch the spacing on each indented line, DevOps expected 2 spaces per indentation level.